### PR TITLE
Wallet prompt interface

### PR DESCRIPTION
**Pull request by ricardoak25**
Show wallet information in the prompt

### CODE REVIEW

File: src/NeoSharp.Application/Client/Prompt[Wallet].cs [hunk 1]
```diff
@@ -16,11 +16,10 @@ private void WalletCreateCommand(string fileName)
                 SecureString secureString = _consoleReader.ReadPassword();
                 _walletManager.CreateAccount(secureString);
             }
-            catch(System.Exception exception)
+            catch (System.Exception exception)
```
Comment (shargon) [inline src/NeoSharp.Application/Client/Prompt[Wallet].cs]: Remove this try /catch (is generic and is captured by Prompt)

File: src/NeoSharp.Application/Client/Prompt[Wallet].cs [hunk 2]
```diff
@@ -40,9 +40,11 @@ private void WalletOpenCommand(string fileName)
...
         {
-            try{
+            try
+            {
                 _walletManager.Close();
-            }catch (System.Exception exception)
+            }
+            catch (System.Exception exception)
             {
                 _consoleWriter.WriteLine(exception.Message, ConsoleOutputStyle.Error);
```
Comment (shargon) [inline src/NeoSharp.Application/Client/Prompt[Wallet].cs]: same

File: src/NeoSharp.Application/Client/Prompt[Wallet].cs [hunk 3]
```diff
@@ -31,7 +30,8 @@ private void WalletOpenCommand(string fileName)
                 _walletManager.Load(fileName);
                 SecureString secureString = _consoleReader.ReadPassword();
                 _walletManager.UnlockAllAccounts(secureString);
-            }catch (System.Exception exception)
+            }
+            catch (System.Exception exception)
             {
```
Comment (shargon) [inline src/NeoSharp.Application/Client/Prompt[Wallet].cs]: same here

**Comment by ricardoak25** [comment]
Removed try catch codes on prompt

File: src/NeoSharp.Application/Client/Prompt[Wallet].cs [hunk 4]
```diff
@@ -10,70 +10,47 @@ public partial class Prompt : IPrompt
...
         [PromptCommand("wallet close", Category = "Wallet", Help = "Close wallet")]
         private void WalletCloseCommand()
         {
-            try{
-                _walletManager.Close();
-            }catch (System.Exception exception)
-            {
-                _consoleWriter.WriteLine(exception.Message, ConsoleOutputStyle.Error);
-            }
+            _walletManager.Close();
```
Comment (shargon) [inline src/NeoSharp.Application/Client/Prompt[Wallet].cs:27]: _walletManager?.Close();

File: src/NeoSharp.Application/Client/Prompt[Wallet].cs [hunk 5]
```diff
@@ -10,70 +10,47 @@ public partial class Prompt : IPrompt
...
-                _consoleWriter.WriteLine(exception.Message, ConsoleOutputStyle.Error);
-            }
+            SecureString secureString = _consoleReader.ReadPassword();
+            _walletManager.ImportEncryptedWif(nep2key, secureString);
+        }
+
+        [PromptCommand("wallet", Category = "Wallet", Help = "List all accounts from wallet")]
+        private void WalletListAccountCommand(PromptOutputStyle output = PromptOutputStyle.json)
+        {
+            IWallet currentWallet = _walletManager.Wallet;
```
Comment (shargon) [inline src/NeoSharp.Application/Client/Prompt[Wallet].cs]: Maybe we can check if is opened for show the user another exception, not NullException

File: src/NeoSharp.Core/Wallet/NEP6/NEP6WalletManager.cs [hunk 6]
```diff
@@ -46,7 +46,13 @@ public void CreateWallet(string filename)
                 throw new ArgumentException("File already exists");
             }
 
-            string walletName = filename.Split('.')[0];
+            string walletName = filename;
+            if (walletName.Contains("/")) {
```
Comment (shargon) [inline src/NeoSharp.Core/Wallet/NEP6/NEP6WalletManager.cs]: use `new FileName()` , Windows users use '\\'

File: src/NeoSharp.Application/Client/Prompt[Wallet].cs (hunk 4 above)
Comment (melanke) [reply src/NeoSharp.Application/Client/Prompt[Wallet].cs:27]: @shargon _walletManager is injected, do we really need to check if it's null in all usages?

**Comment by ricardoak25** [comment]
Its done.

**Pull request merged**